### PR TITLE
fix(command_result): return entire result in toJSON

### PR DESCRIPTION
**Pull request by kvwalker**
Fixes NODE-1942

### CODE REVIEW

File: lib/connection/command_result.js [hunk 1]
```diff
@@ -19,7 +19,9 @@ var CommandResult = function(result, connection, message) {
  * @return {object}
  */
 CommandResult.prototype.toJSON = function() {
-  return this.result;
+  let result = this;
```
Comment (daprahamian) [inline lib/connection/command_result.js]: I think we should be making a copy here. Deleting the message b/c of a toJSON call seems like an unintended side effect.

Comment (kvwalker) [reply lib/connection/command_result.js]: Good catch, thanks!

**Comment by kvwalker** [comment]
I'm a fan of #2. I think that makes the most sense!

**Comment by kvwalker** [comment]
I added https://github.com/mongodb/node-mongodb-native/pull/1977 to test this.

**Comment by mbroadst** [comment]
@kvwalker I was holding off on LGTM'ing this because the native PR is all red in CI, specifically related to checking the result of an `insertOne`

**Comment by kvwalker** [comment]
The native PR is showing that the way `toJSON` currently works is broken so that PR will be green after this PR is merged.

**Comment by mbroadst** [comment]
😱 well don't I feel foolish..

**Pull request merged**